xtask/src/cargo.rs

```diff
@@ -219,7 +232,14 @@ impl Cargo {
         }
 
         if let Some(target) = self.target {
-            cmd.args(["--target", target.as_triple(), "-Zbuild-std=core,alloc"]);
+            cmd.args(["--target", target.as_triple()]);
+
+            // If the target is not installed, use build-std. Keep this
+            // around until our minimum-supported nightly version is at
+            // least 2022-11-10.
+            if !is_target_installed(target.as_triple())? {
```

Comment:
can you create a follow-up ticket please to remove this eventually?

Reply:
I went ahead and made a PR: https://github.com/rust-osdev/uefi-rs/pull/585

We can merge that after Feb 22, 2023.